docs/function-after-focusing-screen.md

```diff
@@ -23,6 +23,8 @@ For instance, if we are attempting to make an API call on focus to fetch some da
 
 ### Example
 
+#TODO  - ldt's make it hoop
```

Comment:
We have 2 hooks:

- `useFocusEffect`: useful to run effects only when screen is focused, for example, you could attach event listeners, connect to socket etc. with this hook
- `useIsFocused`: this returns the current focus state as a boolean. useful for rendering stuff based on the focus state.

We can copy more detailed descriptions from the current README.

Apart from these hooks, we can also document how to add listeners explicitly:

```js
useEffect(() => navigation.addListener('focus', () => { /* do stuff */ }), []);
useEffect(() => navigation.addListener('blur', () => { /* do stuff */ }), []);
```

As well as `navigation.isFocused()`. But mention that users should prefer using the hooks instead of adding the listeners manually.